docs/src/index.md

```diff
@@ -202,3 +202,350 @@ julia> # Again, the optinal chunk size argument allows us to use vector forward
  -400.0  200.0
     2.0    1.0
 ```
+
+## Caveats / Known-issues
+
+### Activity of temporary storage / Activity Unstable Code
+
+If you pass in any temporary storage which may be involved in an active computation to a function you want to differentiate, you must also pass in a duplicated temporary storage for use in computing the derivatives. For example, consider the following function which uses a temporary buffer to compute the result.
+
+```jldoctest storage
+function f(x, tmp, k, n)
+    tmp[1] = 1.0
+    for i in 1:n
+        tmp[k] *= x
+    end
+    tmp[1]
+end
```

Comment:
LGTM except you'll need to rebase docs changes because they were moved to an FAQ page.

Reply:
Oh no, I messed that up... but now fixed.